frontend/layouts/default.vue

```diff
@@ -93,6 +96,9 @@ export default Vue.extend({
         stagingBannerURL() {
             return process.env.STAGING_BANNER_URL;
         },
+        isDev() {
+            return this.$nuxt?.context?.isDev || false;
+        },
```

Comment:
> only if you want to make a story

That's exactly why I did it 😄 But if it's unnecessary I'll change it.

Reply:
And why is it `|| false`? Is it ever undefined? Have you tested that it actually works as intended in production?

Reply:
> And why is it || false?

Uhm good question 😅 It isn't actually required since it never is undefined.
And yes I have tested both dev and production environments and it works as intended